contrib/twitchtv/twirp/twirp.go

```diff
@@ -0,0 +1,208 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+// Package twirp provides tracing functions for tracing clients and servers generated
+// by the twirp framework (https://github.com/twitchtv/twirp).
+package twirp // import "gopkg.in/DataDog/dd-trace-go.v1/contrib/twitchtv/twirp"
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"net/http"
+	"os"
+	"strconv"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/twitchtv/twirp"
+)
+
+type contextKey int
+
+const (
+	twirpErrorKey contextKey = iota
+)
+
+// HTTPClient is duplicated from twirp's generated service code.
```

Comment:
Would it be better to explain how it is used, rather than why it is here?

Reply:
It's not something a user of this integration will use directly, so I think it's fair to focus on the "what" and "why" instead of usage.